src/server-extension/resolvers/spotlight.ts

```diff
@@ -24,28 +26,47 @@ enum OrderDirection {
 export class SpotlightResolver {
   constructor(private tx: () => Promise<EntityManager>) { }
 
-  // TODO: calculate score sold * (unique / total)
   @Query(() => [SpotlightEntity])
   async spotlightTable(
-    @Arg('limit', { nullable: true, defaultValue: null }) limit: number,
-    @Arg('offset', { nullable: true, defaultValue: null }) offset: string,
-    @Arg('orderBy', { nullable: true, defaultValue: 'total' }) orderBy: OrderBy,
-    @Arg('orderDirection', { nullable: true, defaultValue: 'DESC' }) orderDirection: OrderDirection
+      @Arg('limit', { nullable: true, defaultValue: null }) limit: number,
+      @Arg('offset', { nullable: true, defaultValue: null }) offset: string,
+      @Arg('orderBy', { nullable: true, defaultValue: 'total' }) orderBy: OrderBy,
+      @Arg('orderDirection', { nullable: true, defaultValue: 'DESC' }) orderDirection: OrderDirection,
+      @Arg('passionList', { nullable: true, defaultValue: null }) passionList: string
   ): Promise<SpotlightEntity[]> {
+    let tmpPassionList: string[] = []
+    try {
+      tmpPassionList = JSON.parse(passionList)
+      if (!Array.isArray(tmpPassionList)) {
+        throw new Error("not array")
+      }
+      if (tmpPassionList.some(passion => typeof passion !== "string")) {
+        throw new Error("not string array")
+      }
+    } catch {
+      console.error("Invalid passionList")
```

Comment:
hey, I'm wondering how should this work, I'm passing in various addresses and still ending with "Invalid passionList" in console